traits.yml

```diff
@@ -235,6 +496,45 @@ traits:
           Manager.getWithRelationshipPaged.
         usage:
           - relationship
+  representation:
+    description: >
+      Traits relating to alternate representations of what is otherwise
+      logically the same entity.
+    members:
+      Proxy:
+        description: >
+          An alternate scale/quality representation. In the case of
+          images, this is usually either a lower resolution, reduced
+          quality encoding. For spatial data this may be reduced
+          spacial fedlity or number of points.
```

Comment:
[Consider]
```suggestion
          images, this is usually a lower resolution, reduced
          quality encoding. For spatial data this may be reduced
          spacial fedlity or number of points.
```

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/71/commits/e4a2bf72e689f6d4ca7841a858931347a0adc511